std/conv.d

```diff
@@ -5653,17 +5665,19 @@ auto toChars(ubyte radix = 10, Char = char, LetterCase letterCase = LetterCase.l
             {
                 Result result = void;
                 result.buf = buf;
-                result.lwr = cast(ubyte)(this.lwr + lwr);
-                result.upr = cast(ubyte)(this.lwr + upr);
+                result.lwr = cast(uint)(this.lwr + lwr);
+                result.upr = cast(uint)(this.lwr + upr);
                 return result;
             }
 
           private:
+            uint lwr, upr;
             char[(UT.sizeof == 4) ? 10 + isSigned!T : 20] buf;
```

Comment:
Wouldn't = void here give us the same effect? Or the compiler will still bit-blit all the whole thing over?